Gemfile

```diff
@@ -1,6 +1,6 @@
 source "http://rubygems.org"
 
-gem 'chef-analytics', git: 'git@github.com:opscode/chef-analytics-api.git'
+gem 'chef-analytics', git: 'git@github.com:chef/chef-analytics-api.git'
```

Comment:
Since this is a public repo, please use the HTTPS transport to ensure it can be easily built without github keys.